src/EditorFeatures/Core/LineSeparators/LineSeparatorsOptions.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.LineSeparators
+{
+    internal static class LineSeparatorsOptions
+    {
+        public static readonly PerLanguageOption2<bool> LineSeparator = new("dotnet_line_separator", defaultValue: false);
```

Comment:
>dotnet_line_separator [](http://example.com/codeflow?start=77&length=21)

How is this different from `end_of_line` editorconfig option? Perhaps we should follow up on unifying these?

Reply:
Oh this is for display line separator.
I will update the name